libs/books/data-access/src/lib/+state/reading-list.effects.ts

```diff
@@ -54,9 +66,74 @@ export class ReadingListEffects implements OnInitEffects {
     )
   );
 
+  undoAddtoReadingList$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(ReadingListActions.confirmedAddToReadingList),
+      filter(({book}) => book.isOpenSnackBar),
+      switchMap(({ book }) => {
+        return of(
+          ReadingListActions.openSnackBar({
```

Comment:
Please remove this action from `reading-list.actions.ts` as we are already using `confirmedAddToReadingList` as a trigger point for snackbar.
Same is applicable for next effect as well.